bsunzip.sh

```diff
@@ -59,9 +59,9 @@ strip_cruft() {
 
 }
 
-# brightspace user comments are put in a quite unusable HTLM file
+# brightspace user comments are put in a quite unusable HTML file
```

Comment:
HTLM wasn't actually a typo, but a reference. But I guess @squell was fine with it...

Reply:
I appearantly was out of the loop for HTLM:')

Reply:
Actually I was just too lazy to split up the PR just  to pay homage to that bit of folklore. :(